### ISSUE TITLE
CASL Angular: Error with 'test' command

### ISSUE THREAD

**Issue by wickedsp1d3r** [stalniy/casl#266]
I get this error on Windows 10 after issuing `yarn upgrade`

```
'test' is not recognized as an internal or external command, operable program or batch file.
```

**Comment by stalniy** [comment]
`@casl/angular@3.0.1` has been published. Please check whether it works in your env

**Comment by stalniy** [comment]
missed important to add utility file, fixed in `@casl/agular@3.0.2`

**Comment by wickedsp1d3r** [comment]
Thank you for the answer. I get a new error:

```
error <path>\node_modules\@casl\angular: Command failed
Exit code: 1
Command ./tools/exists.js package-lock.json && ngcc || true
Arguments:
Directory: <path>\node_modules\@casl\angular
Output:
internal/modules/cjs/loader.js:983
  throw err;
  ^

Error: Cannot find module '<path>\node_modules\@casl\angular\tools\exists.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader:980:15)
    at Function.Module._load (internal/modules/cjs/loader.js:862:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at interal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
```

Also, I have started to see errors like these (I don't know if these are related to this):
```
node_modules/@casl/angular/dist/types/can.d.ts:2:23 - error TS2305 Module '"<path>/node_modules/@casl/angular"' has no exported member 'AnyAbility'
import { Unsubsribe, AnyAbility, AbilityParameters } from '@casl/ability';

node_modules/@casl/angular/dist/types/can.d.ts:2:35 - error TS2305 Module '"<path>/node_modules/@casl/angular"' has no exported member 'AbilityParameters'
import { Unsubsribe, AnyAbility, AbilityParameters } from '@casl/ability';

node_modules/@casl/angular/dist/types/module.d.ts:2:10 - error TS2305 Module '"<path>/node_modules/@casl/angular"' has no exported member 'AnyAbility'
import { AnyAbility } from '@casl/ability';
```

**Comment by stalniy** [comment]
Yes, it’s related. Please try to upgrade to the latest version of @casl/angular

**Comment by stalniy** [comment]
Fixed in `@casl/angular@3.0.5`, tested on Windows 7 all good
